tests/test_suite.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python2
+from __future__ import print_function
+
+import argparse
+import os
+import os.path
+import sys
+import time
+
+from lib.log import log
+import lib.log
+import lib.oscap
+import lib.virt
+import lib.profile
+import lib.rule
+
+parser = argparse.ArgumentParser()
+
+common_parser = argparse.ArgumentParser(add_help=False)
+common_parser.add_argument("--hypervisor",
+                           dest="hypervisor",
+                           metavar="qemu:///HYPERVISOR",
+                           default="qemu:///session",
+                           help="libvirt hypervisor")
+common_parser.add_argument("--domain",
+                           dest="domain_name",
+                           metavar="DOMAIN",
+                           default=None,
+                           help="libvirt domain used as test bed")
```

Comment:
We should describe what happens to the domain. Does it get changed? Can it get bricked? How does the snapshotting work?